smurfs/src/actions/index.js

```diff
@@ -1,15 +1,43 @@
-/* 
-  Action Types Go Here!
-  Be sure to export each action type so you can pull it into your reducer
-*/
+import axios from "axios";
+
+export const FETCH_SMURF = "FETCH_SMURF";
+export const CREATE_SMURF = "CREATE_SMURF";
+export const ADDING_SMURF_SUCC = "ADDING_SMURF_SUCC";
+export const ADDING_SMURF_FAIL = "ADDING_SMURF_FAIL";
+export const DELETING_SMURF = "DELETING_SMURF";
+
+export const getSmurfs = () => dispatch => {
+  dispatch({ type: FETCH_SMURF });
+  axios
+    .get('http://localhost:3333/smurfs')
+    .then(res => dispatch({ type: ADDING_SMURF_SUCC, payload: res.data }))
+    .catch(err => dispatch({type: ADDING_SMURF_FAIL, payload: err }))
+}
+export const addSmurf = () => dispatch => {
+  dispatch({ type: CREATE_SMURF }); 
+  axios 
+    .get('http://localhost:3333/smurfs')
```

Comment:
axios.post..... not GET

Reply:
ooh and you didn't pass your state in. remember posting something gets two arguments....endpoint to post to and what it is you want to post.